packages/openapi-v3/test/integration/controller-spec.integration.ts

```diff
@@ -219,7 +219,48 @@ describe('controller spec', () => {
     });
   });
 
-  it('generates schema for an array from `x-ts-type`', () => {
+  @model()
```

Comment:
I find it rather unexpected to encounter a helper model being defined in the middle of the test cases. Please move the class definition to the bottom of this `describe` block.